### DISCUSSION TITLE
签到的URL每天都会变如何解..

### DISCUSSION THREAD

**Discussion by christaikobo** [qd-today/qd#377] [Q&A] [answered]
比如
`https://xxx.xxx.com/*****.php?ok=3&safeid=07785dd`

07785dd每天都会变....

**Comment by a76yyyy** [comment, marked as answer]
很显然, 这是模板变量最基本的用处, 
只需要找到 `safeid` 的出处, 就可以通过动态获取解决

**Reply by christaikobo** [reply]
> 很显然, 这是模板变量最基本的用处, 只需要找到 `safeid` 的出处, 就可以通过动态获取解决

非常感谢作者亲自回答，我并没有找到相关的文档，请问在哪里有说明我可以参考吗？或者是否有现成模板可供参考？
`https://xxx.xxx.com/*****.php?ok=3&safeid=07785dd` 是在一个“请点击这里”的文字附带的链接，是不是可以通过检测“请点击这里”来获取链接

**Reply by christaikobo** [reply]
> > 很显然, 这是模板变量最基本的用处, 只需要找到 `safeid` 的出处, 就可以通过动态获取解决
> 
> 非常感谢作者亲自回答，我并没有找到相关的文档，请问在哪里有说明我可以参考吗？或者是否有现成模板可供参考？ `https://xxx.xxx.com/*****.php?ok=3&safeid=07785dd` 是在一个“请点击这里”的文字附带的链接，是不是可以通过检测“请点击这里”来获取链接

自己研究出来了，在前一个页面的预览界面进行变量提取，content可以去匹配所有html代码的，找到合适的代码提取出变量带入下一页请求里面就可以了

**Reply by a76yyyy** [reply]
公共模板有很多可以借鉴的，可以参考
